scripts/setup_for_crt_builder.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -x
+
+# Generates the Package.swift file, configured to work properly when using the
+# CRT builder script.
+# For use only as a pre-build step when using the CRT builder script on CI.
```

Comment:
What does this script have to do with CRT or the CRT builder script? From this PR it appears we're using this script as part of the aws-sdk-swift builder script.

Reply:
When we run CI tasks, you'll see that the 1st thing we do is retrieve and install a script from a Cloudfront distro.  That is the "CRT builder script" I refer to.  (It's on Github [here](https://github.com/awslabs/aws-crt-builder).)

That script installs `aws-sdk-swift`, `smithy-swift`, and `aws-crt-swift` to the local filesystem.  Env vars in the CI job point to the specific locations on the filesystem.  

This script rewrites the `Package.swift` locally to require `smithy-swift` and `aws-crt-swift` at the locations in those env vars.

Reply:
ahhh TIL thank you. 
it might be helpful to include the link to that repo in the comments here

Reply:
I will add the link to the build script repo to the comments in this script.

To be clear, we don't _need_ the CRT builder script to build any of our projects on CI, but most of our CI tasks are currently based on it (not the lint tasks).  Ripping it out will be a project for another day.